### PR TITLE
[DBZ-1073] Propagate column type scale

### PR DESCRIPTION
**Pull request by amitsela**
This PR adds the scale, if exists, to the column type propagation.

### CODE REVIEW

File: debezium-core/src/main/java/io/debezium/relational/mapping/PropagateSourceTypeToSchemaParameter.java [hunk 1]
```diff
@@ -19,10 +19,11 @@
  * @author Orr Ganani
  * @author Gunnar Morling
  */
-public class PropagateSourceTypeToSchemaParameter implements ColumnMapper {
+public class  PropagateSourceTypeToSchemaParameter implements ColumnMapper {
```
Comment (gunnarmorling) [inline debezium-core/src/main/java/io/debezium/relational/mapping/PropagateSourceTypeToSchemaParameter.java]: Can you revert this one?

**Comment by gunnarmorling** [comment]
@amitsela And could you also provide a PR for the website repo to add the new option to the docs? Thanks!

File: debezium-core/src/main/java/io/debezium/relational/mapping/PropagateSourceTypeToSchemaParameter.java (hunk 1 above)
Comment (amitsela) [reply debezium-core/src/main/java/io/debezium/relational/mapping/PropagateSourceTypeToSchemaParameter.java]: Oops, sure!

**Comment by amitsela** [comment]
retest this please

**Comment by amitsela** [comment]
squashed

**Comment by amitsela** [comment]
not sure why it fails now since I only removed the redundant whitespace. Don't think I have permissions to rebuild Travis..

**Comment by gunnarmorling** [comment]
Applied. Thanks a lot, @amitsela!

**Comment by amitsela** [comment]
Happy to help!

**Pull request merged**